app/components/NewColorsInput/NewColorsInput.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ColorInput from '../ColorInput/ColorInput';
+
+export default function NewColorsInput({
+  newColorA: A,
+  newColorB: B,
+  newColorC: C,
+  updateNewColor,
+}) {
+  const colors = [
+    { colorLetter: 'A', currentNewColor: A, defaultText: 'red (#f00)', },
+    { colorLetter: 'B', currentNewColor: B, defaultText: 'blue (#00f)', },
+    { colorLetter: 'C', currentNewColor: C, defaultText: 'green (#0f0)', },
+  ];
+  function createNewColorInput(color, index) {
+    const { colorLetter, currentNewColor, defaultText, } = color;
+    const colorInputProps = {
+      colorLetter,
+      currentNewColor,
+      defaultText,
+      key: index,
```

Comment:
Assuming the colors are unique, they would make a better index. Read about keys in react docs.

Reply:
Indeed a good point. I automatically dismissed all the other data since I didn't have any id passed in, but the `colorLetter` will be a great key in this case.

Reply:
done